x-pack/plugins/aiops/public/components/change_point_detection/change_points_table.tsx

```diff
@@ -126,6 +137,56 @@ export const ChangePointsTable: FC<ChangePointsTableProps> = ({
             truncateText: false,
             sortable: true,
           },
+          {
+            name: i18n.translate('xpack.aiops.changePointDetection.actionsColumn', {
+              defaultMessage: 'Actions',
+            }),
+            actions: [
```

Comment:
As discussed, if it isn't too much work, I think it would be good to add a 'Filter out' action for adding a `NOT` style filter, and use the plus / minus icons as used in Discover and Lens.

<img width="766" alt="image" src="https://user-images.githubusercontent.com/7405507/233332781-c57e4997-db78-4bac-8744-9805fec0bec7.png">

Reply:
Updated in https://github.com/elastic/kibana/pull/155256/commits/d0600fbe024780aa93c375d507b4f4f0501ded56